### ISSUE TITLE
[Download improvements]Choose download path before starting download,Pause/resume

### ISSUE THREAD

**Issue by Mitia99** [tuyafeng/Via#644]
I wonder if you could add an option to set download path before the download starts and add a pause/resume(and indication if the download is resumable or not) feature.Thanks

**Comment by tuyafeng** [comment]
I will improve this part. The main reason for this situation is that I need to adapt to android11 ​​first.

**Comment by tuyafeng** [comment]
Duplicate of #536
